packages/plugins/ScamSniffer/src/base.ts

```diff
@@ -0,0 +1,17 @@
+import type { Plugin } from '@masknet/plugin-infra'
+import { PLUGIN_DESCRIPTION, PLUGIN_ID, PLUGIN_NAME } from './constants'
+import { languages } from './locales/languages'
+
+export const base: Plugin.Shared.Definition = {
+    ID: PLUGIN_ID,
+    name: { fallback: PLUGIN_NAME },
+    description: { fallback: PLUGIN_DESCRIPTION },
+    publisher: { name: { fallback: 'Scam Sniffer' }, link: 'https://scamsniffer.io/' },
+    enableRequirement: {
+        architecture: { app: false, web: true },
+        networks: { type: 'opt-out', networks: {} },
```

Comment:
Does it work for all networks? if not, please specify the network you supports.

Reply:
yeah, It's not depends on any network